src/locale/test-keymap-util.c

```diff
@@ -93,9 +94,13 @@ static void test_vconsole_convert_to_x11(void) {
         log_info("/* test with known variant, new mapping (es:dvorak) */");
         assert_se(free_and_strdup(&c.vc_keymap, "es-dvorak") >= 0);
 
-        assert_se(vconsole_convert_to_x11(&c) == 0); // FIXME
+        r = vconsole_convert_to_x11(&c);
+        assert_se(r >= 0);
         assert_se(streq(c.x11_layout, "es"));
-        assert_se(c.x11_variant == NULL); // FIXME: "dvorak"
+        if (r == 0)
+                assert_se(c.x11_variant == NULL); // FIXME: "dvorak"
```

Comment:
Maybe, it is ok to drop `FIXME` comment now?
Also, please use the implicit cast from pointer to boolean:
```
assert_se(!c.x11_variant);
```

Reply:
AFAICS src/locale/kbd-model-map still doesn't have a mapping for es-dvorak therefore the test would fail with the default kbd-model-map.

Reply:
Shouldn't we add the mapping then?

Reply:
Or, at least, could you update the comment to elaborate more?

Reply:
I've no idea how this kbd-model-map file is supposed to be maintained. According to its header it's been generated by `system-config-keyboard`, which I assume is a tool from Fedora.

I also assumed that there was a reason why the author of this test preferred to add a workaround in the test rather than updating kbd-model-map directly.

Reply:
It **was** generated. Now it's being maintained here. Maybe add "originally" in the comment in that file?

Reply:
ok

Reply:
system-config-keyboard was a fedora project, hosted at https://fedorahosted.org/system-config-keyboard/. fedorahosted.org has been shut down a while back (https://fedoraproject.org/wiki/Infrastructure/Fedorahosted-retirement), but s-c-k was not migrated to pagure.io. Afaict, it's gone. The Fedora package was unmaintained and retired two years ago (https://src.fedoraproject.org/rpms/system-config-keyboard/c/01411c1da57bf87610d766e3c00e9242d2c6fb96?branch=rawhide). I think it was python2-only too.